host-configs/llnl-toss3-MI100-rocm4-whamo.cmake

```diff
@@ -20,15 +20,16 @@ set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-std=c++14 -O3 -g -ffast-math" CACHE STRING
 set(CMAKE_CXX_FLAGS_DEBUG "-std=c++14 -O0 -g" CACHE STRING "")
 
 set(ENABLE_CHAI On CACHE BOOL "")
-set(CHAI_ENABLE_RAJA_PLUGIN On CACHE BOOL "")
-set(ENABLE_RAJA_PLUGIN On CACHE BOOL "")
 set(ENABLE_HIP On CACHE BOOL "")
 set(ENABLE_OPENMP Off CACHE BOOL "")
-set(ENABLE_MPI_WRAPPER Off CACHE BOOL "")
+set(ENABLE_MPI Off CACHE BOOL "")
 
 #set(CMAKE_HIPCC_FLAGS_RELEASE "-O3 --expt-extended-lambda" CACHE STRING "")
 #set(CMAKE_HIPCC_FLAGS_RELWITHDEBINFO "-O3 -lineinfo --expt-extended-lambda" CACHE STRING "")
 #set(CMAKE_HIPCC_FLAGS_DEBUG "-O0 -g -G --expt-extended-lambda" CACHE STRING "")
 #set(CMAKE_HIPCC_HOST_COMPILER "${CMAKE_CXX_COMPILER}" CACHE STRING "")
 
+# For LLNL TCE packages
+set(ENABLE_MPI_WRAPPER Off CACHE BOOL "")
```

Comment:
@vsrana01 I've made `ENABLE_MPI` the more default option so that our configs are more portable, so that we might avoid problems like this https://github.com/LLNL/Kripke/issues/34. If someone wants to use our TCE packages, they can turn on `ENABLE_MPI_WRAPPER`. Let me know if this is ok with you.